fairscale/optim/adascale.py

```diff
@@ -0,0 +1,233 @@
+# Copyright 2020 Petuum, Inc. All Rights Reserved.
```

Comment:
Since the license is the same as the root LICENSE file, I wonder can you just replace this with these lines?

![image](https://user-images.githubusercontent.com/24926999/96195530-d651c880-0f01-11eb-822c-af955de2269d.png)

Reply:
Copyright Facebook is not necessary. We only add a Facebook copyright when an FB employee modifies the file, always preserving the original copyright.